_episodes/04-data-types-and-format.md

```diff
@@ -15,7 +15,11 @@ objectives:
     - "Analyze datasets having missing/null values (NaN values)."
     - "Write manipulated data to a file."
 keypoints:
-    - "FIXME"
+    - "Pandas and base Python use slightly different names for data types"
+    - "Each column in a DataFrame has a shared data type for all rows"
+    - "The type of one column in a DataFrame can be checked using `dataFrameName[column_name].dtype`"
+    - "Make conscious decisions about how to manage missing data"
+    - "A DataFrame can be saved to a CSV file using `pd.read_csv`"
```

Comment:
read?

Reply:
also, let's not us `pd.<something>` but `Panda's <something>`

Reply:
Agree, perhaps:

"A DataFrame can be saved to a CSV file using the `read_csv` function."

Reply:
My points were that 1. the command to write dataframe to a file is `to_csv` and 2. `pd` is not introduced in keypoints so is someone reads through keypoints only they will have hard time understanding this. Also, at least in swcarpentry/python-novice-inflammation lesson there was a decision to use full module names (`numpy` and not `np`, `matplotlib.pyplot` and not `plt`). `pd` is a useful shortcut but unnecessary here.